src/inbox/mod_inbox_rdbms.erl

```diff
@@ -61,6 +63,20 @@ get_inbox_rdbms(LUser, LServer, #{ order := Order } = Params) ->
                  " ORDER BY timestamp ", OrderSQL, ";"],
     mongoose_rdbms:sql_query(LServer, Query).
 
+
+get_inbox_unread(Username, Server) ->
+    Res = mongoose_rdbms:sql_query(Server,
+                                  ["select unread_count from inbox "
+                                   "WHERE luser=", esc_string(Username), 
+                                   " AND lserver=", esc_string(Server), ";"]),
+   case check_result(Res) of
+       {ok, []} ->
+          ok;
+       {ok, Count} ->
+           {ok, Count + 1}
```

Comment:
I think it's kind of unexpected, isn't it? Backend should return current unread count, not some incremented value.